internal/api/commitment.go

```diff
@@ -126,8 +126,8 @@ var (
 		 WHERE ps.project_id = $1 AND ps.type = $2 AND pr.name = $3 AND par.az = $4
 	`)
 	forceImmediateCapacityScrapeQuery = sqlext.SimplifyWhitespace(`
-		UPDATE cluster_capacitors SET next_scrape_at = $1 WHERE capacitor_id = (
-			SELECT capacitor_id FROM cluster_services cs JOIN cluster_resources cr ON cs.id = cr.service_id
+		UPDATE cluster_services SET next_scrape_at = $1 WHERE id = (
+			SELECT cs.id FROM cluster_services cs JOIN cluster_resources cr ON cs.id = cr.service_id
 			WHERE cs.type = $2 AND cr.name = $3
 		)
```

Comment:
This can be simplified now using the 1:1 relation between capacitors and services:
```suggestion
		UPDATE cluster_services SET next_scrape_at = $1 WHERE type = $2
```
Also, since the query is only a short oneliner now, it can probably be inlined, too.